.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
```

Comment:
Now VSCode users will enjoy automatic code formatting when they hit save (provided they have the ESLint plugin installed).

Reply:
Nice! I was wondering how to do that.